snippets/csharp/System.Windows.Forms/FolderBrowserDialog/Overview/form1.cs

```diff
@@ -1,4 +1,3 @@
-﻿//<Snippet1>
```

Comment:
This line needs to be included. It's used by the OPS build system to render the sample.

Reply:
Woops! Sorry, didn't mean to delete that. I'll fix it right away.